flimfit.html

```diff
@@ -67,7 +67,8 @@
 
 <ol class="step">
 
-<li><p class="first">Download the latest release of FLIMfit from:</p>
+<li><p class="first">Use the link below to go to the FLIMfit downloads page.</p>
+<p>Click on the link under <span class="bold">Latest Version</span> to go to the downloads page for that version.</p>
 
 <p><a href="http://downloads.openmicroscopy.org/latest/flimfit" target="_blank" class="external-url">http://downloads.openmicroscopy.org/latest/flimfit</a></p>
```

Comment:
I think http://flimfit.org/downloads/ is better. Have pushed commit to do that but happy to change if decided.